src/Microsoft.Data.SqlClient/tests/FunctionalTests/LocalizationTest.cs

```diff
@@ -0,0 +1,177 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Data;
+using System.Data.Common;
+using System.Globalization;
+using System.Reflection;
+using System.Security;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.SqlServer.TDS.Servers;
+using Xunit;
```

Comment:
Remove unnecessary usings.

Reply:
Removed unnecessary usings and sorted the rest.